storybookjs/storybook
# PanelTabOptions WIP

Issue: 
no options to customize panel tabs. similar to `showPanel`, allow user to config Panel tabs globally or per story.
## What I did
- added two options to layout api, `hiddenTabs` & `tabsOrder` (names up for discussion), in order to customize panel tabs.
- Add sorter method within panel component

***Note, having trouble building (on windows), will rely on CI for now , therefore this PR may be temporary.
 
## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
maybe?
- Does this need an update to the documentation?
yep

If your answer is yes to any of these, please make sure to include it in your PR.
